packages/globe_cli/lib/src/commands/build_logs_command.dart

```diff
@@ -39,24 +36,25 @@ class BuildLogsCommand extends BaseGlobeCommand {
       return ExitCode.software.code;
     }
 
-    late Stream<BuildLogEvent> logs;
+    final deployment = await api.getDeployment(
+      orgId: validated.organization.id,
+      projectId: validated.project.id,
+      deploymentId: deploymentId,
+    );
 
-    try {
-      logs = await streamBuildLogs(
-        api: api,
-        orgId: validated.organization.id,
-        projectId: validated.project.id,
-        deploymentId: deploymentId,
-      );
-    } on EventSourceSubscriptionException catch (e) {
-      logger.err(
-        e.statusCode == HttpStatus.notFound
-            ? '✗ An error occurred: Either invalid deployment or project ID. '
-            : '✗ An error occurred: ${e.message}',
-      );
+    if (deployment.buildId == null) {
+      logger.err('Build for $deploymentId has not started yet.');
```

Comment:
Small nit; does it make sense for this to be more of a informational message or error?